podpac/core/data/datasource.py

```diff
@@ -281,6 +281,24 @@ def _get_data(self, rc, rci):
     # Methods
     # ------------------------------------------------------------------------------------------------------------------
 
+    def get_source_data(self, bounds={}):
+        """
+        Get source data, without interpolation.
+
+        Arguments
+        ---------
+        bounds : dict
+            Dictionary of bounds by dimension, optional.
```

Comment:
It looks like the bound have to be in the format `{"dim", (min, max)}` and `(max,min)` does not work). Is that correct?

Reply:
cc59d811